core/db/migrate/20250604072555_add_fk_to_product_properties.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddFkToProductProperties < ActiveRecord::Migration[7.0]
+  def change
+    add_foreign_key :spree_product_properties, :spree_products, column: :product_id, null: false
```

Comment:
Same here, if there are rows without these constraints respected, will the migration fail?

Reply:
Same as above.